modin/pandas/test/test_general.py

```diff
@@ -696,7 +696,6 @@ def test_to_pandas_indices():
     [
         (lambda df: df.mean(level=0), r"DataFrame\.mean"),
         (lambda df: df + df, r"DataFrame\.add"),
-        (lambda df: df.index, r"DataFrame\.get_axis\(0\)"),
```

Comment:
why are we removing a testcase?

Reply:
This test is no longer needed because we changed the _get_axis for BaseOnPython.

Reply:
have we added the test for `_get_axis` in base implementation then?